rules/CodeQuality/Rector/MethodCall/AssertCompareOnCountableWithMethodToAssertCountRector.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\PHPUnit\CodeQuality\Rector\MethodCall;
+
+use Countable;
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Expr\StaticCall;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Name;
+use PHPStan\Type\ObjectType;
+use Rector\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see \Rector\PHPUnit\Tests\CodeQuality\Rector\MethodCall\AssertCompareOnCountableWithMethodToAssertCountRector\AssertCompareOnCountableWithMethodToAssertCountRectorTest
+ */
+class AssertCompareOnCountableWithMethodToAssertCountRector extends AbstractRector
+{
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition('', [
+            new CodeSample(
+                <<<'CODE_SAMPLE'
+$this->assertSame(1, $countable->count());
+CODE_SAMPLE
+                ,
+                <<<'CODE_SAMPLE'
+$this->assertCount(1, $countable);
+CODE_SAMPLE
+            ),
+        ]);
+    }
+
+    /**
+     * @return array<class-string<MethodCall|StaticCall>>
+     */
+    public function getNodeTypes(): array
+    {
+        return [MethodCall::class, StaticCall::class];
+    }
+
+    /**
+     * @param MethodCall|StaticCall $node
+     */
+    public function refactor(Node $node): MethodCall|StaticCall|null
+    {
```

Comment:
call is in test case class should be checked, like other rule

https://github.com/rectorphp/rector-phpunit/blob/acfcef8cfbceefee966253ef5f1b2ec57bc10e22/rules/CodeQuality/Rector/MethodCall/AssertComparisonToSpecificMethodRector.php#L94